x/ethbridge/keeper/keeper.go

```diff
@@ -75,7 +75,7 @@ func (k Keeper) ProcessSuccessfulClaim(ctx sdk.Context, claim *types.EthBridgeCl
 		err = k.bankKeeper.MintCoins(ctx, types.ModuleName, coins)
 	case types.ClaimType_CLAIM_TYPE_BURN:
 		coins = sdk.Coins{sdk.NewCoin(tokenMetadata.Symbol, claim.Amount)}
-		err = k.bankKeeper.MintCoins(ctx, types.ModuleName, coins)
+		err = nil
```

Comment:
We can't just do nothing. We have to transfer from module to the account.

Reply:
code line 95	if err := k.bankKeeper.SendCoinsFromModuleToAccount(
will transfer the coin.

Reply:
Wouldn't this double spend coins in the CLAIM_TYPE_MINT because the user gets minted coins on line 75, then on line 95, we then transfer coins from the module to this user.

Reply:
mint just for module account, which get the minted token, then transfer to reciver account.